Data acquisition chapter re-write (#436)
worldbank/dime-data-handbook: chapters/data-collection.tex

%------------------------------------------------

\begin{fullwidth}
High-quality data is essential to most modern development research.
Many research questions require novel data, 
because no source of publicly available data addresses the 
inputs or outcomes of interest for the relevant population.
Data acquisition can take many forms, including: 
primary data generated through surveys; 
private sector partnerships granting access to new data sources, such as administrative and sensor data;

> digitization of paper records, including administrative data;

Reviewer: ```suggestion
digitization of paper records, including administrative data; web scraping;
```
I think this should be a category of its own here.

The second section goes into detail on ensuring data quality if you are collecting survey data.
It provides detailed guidance on the electronic survey workflow,
from questionnaire design to programming and monitoring data quality.
We conclude with a discussion of safe data handling, 
providing guidance on how to receive, transfer, store, and share confidential data.
Secure file management is a basic requirement to comply with the legal and 
ethical agreements that allow  access to personal information for research purposes.

\end{fullwidth}

%------------------------------------------------
\section{Acquiring data}
Clearly establishing and documenting data access is critical for reproducible research.  
When acquiring data, the first step is to determine who owns the data, 
and establish and document the research team's right to use the data. 
Except in the case of primary surveys funded by the research team, 
the data is typically not owned by the research team. 
Rather, they must enter into data licensing agreements 
to access the data and publish derivative work. 
These agreements should make clear from the outset whether the 

> research team can make the original data, or any portion thereof, public. 

Reviewer: ```suggestion
research team can make the original data, any portion thereof, or derivatives\sidenote{
  \textbf{Derivatives} of a data can be indicators, aggregates etc. based on the data.} 
 of the data, public. 
```
Don't know if this is the best way to do it, but I think we should include derivatives already here.

Rather, they must enter into data licensing agreements 
to access the data and publish derivative work. 
These agreements should make clear from the outset whether the 
research team can make the original data, any portion thereof, or derivatives\sidenote{
	\textbf{Derivatives} of a data can be indicators, aggregates etc. based on the data.} 
of the data, public. 

Data ownership\sidenote{need to add DIME Wiki page} can be challenging to establish,
as many jurisdictions have differing laws regarding data and information,
and the research team may be subject to multiple conflicting regulations.
In some countries, data is implicitly owned by the people who it is about.
In others, it is owned by the people who collected it.
In still more, it is highly unclear and there are varying norms.
The best approach is always to consult with a local partner, 
and enter into specific legal agreements establishing ownership, 
access, and publication rights.
This is particularly critical where confidential data is involved
-- that is, when people are disclosing information to you
that you could not obtain simply by observation or through public records. 

> For data generated for the research study, such as survey data, 

Reviewer: ```suggestion
For primary data generated by the research team, such as survey data, 
```
This builds better on vocabulary we have already established

it is important to clarify up front who owns the data,
and who will have access to the data.
These details need to be shared with respondents when they are offered the opportunity 
to consent to participate in the study. 

> If the research team is not collecting survey data directly --

Reviewer: ```suggestion
If the research team is not collecting the data directly --
```
Not specific to survey data

Similarly, when you own a dataset,
you must consider whether you will make the dataset accessible to other researchers,
and what terms-of-use you require. 

If the research team requires access to existing data for novel research,
terms of use should be agreed on with the data owner, 
typically through a data licensing agreement\sidenote{need to add a DIME wiki page}.
Keep in mind that the data owner is likely not familiar
with the research process, and therefore may be surprised
at some of the things you want to do if you are not clear up front.
You will typically want intellectual property rights to all research outputs developed used the data,
a license for all uses of derivative works, including public distribution 
(unless ethical considerations contraindicate this).
This is important to allow the research team to store, catalog, and publish, in whole or in part,
either the original licensed dataset or datasets derived from the original.
Make sure that the license you obtain from the data owner allows these uses,
and that you consult with the owner if you foresee exceptions with specific portions of the data.

> The World Bank has a template Data License Agreement which DIME follows.\sidenote{ \url{https://worldbankgroup.sharepoint.com/teams/ddh/SiteAssets/SitePages/ddh/DataLicenseAgreementTemplate_v4.pdf?cid=68a54269-bbff-4b47-846d-cab248ad7de1}}

Reviewer: ```suggestion
The World Bank has a template Data License Agreement which DIME follows.\sidenote{
\url{https://worldbankgroup.sharepoint.com/teams/ddh/SiteAssets/SitePages/ddh/DataLicenseAgreementTemplate_v4.pdf?cid=68a54269-bbff-4b47-846d-cab248ad7de1}}
```
just a line break so the line is not so wide. We cannot have line breaks before `\sidenote{` as it creates unwanted white space but right afterwards work.

> The World Bank has a template Data License Agreement which DIME follows.\sidenote{ \url{https://worldbankgroup.sharepoint.com/teams/ddh/SiteAssets/SitePages/ddh/DataLicenseAgreementTemplate_v4.pdf?cid=68a54269-bbff-4b47-846d-cab248ad7de1}}

Reviewer: This link is behind WB 2-factor authentication when I tested. Can you open this on a non-WB computer?

Reviewer: Actually, I couldnt even open on my WB computer. I get this message:

![image](https://user-images.githubusercontent.com/15911801/82395767-3e1aea80-9a1a-11ea-8f71-5ac0787c08d7.png)

Author: oh bother. i thought i had found the external link. i now this is now publicly available, will dig to find a link that works.

Reviewer: Did you find the public link?
